src/components/alert.tsx

```diff
@@ -2,14 +2,14 @@ import * as React from 'react';
 import { Box, Flex, ExclamationMarkCircleIcon, CloseIcon } from '@blockstack/ui';
 import { Title, Text, Pre } from '@components/typography';
 import { useClearErrors } from '@common/hooks/use-clear-errors';
-import { useDebugState } from '@common/sandbox';
+import { useSandboxState } from '@common/hooks/use-sandbox-state';
 
 export const renderErrorMessage = ({
   reason,
   reason_data,
   txid,
 }: {
-  reason: 'FeeTooLow' | 'NotEnoughFunds' | 'NoSuchContract';
+  reason: 'FeeTooLow' | 'NotEnoughFunds' | 'NoSuchContract' | 'ContractAlreadyExists' | 'BadNonce';
```

Comment:
I think I prefer using an `Enum` here. This way, when someone calls `renderErrorMessage`, they can pass `{ reason: ErrorReasons.FeeTooLow }`, etc.

It would also be useful for the switch statement a little lower down.